parm/config/gfs/config.resources.GAEA

```diff
@@ -21,6 +20,65 @@ case ${step} in
     esac
     ;;
 
+  "eupd")
+    # update ntasks to 80 and threads_per_task to 20
+    case ${CASE} in
+      "C768")
+        export ntasks=80
+        export threads_per_task=20
+        ;;
+      *)
+        ;;
+    esac
+    export tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
+    ;;
+
+  "analcalc")
+    # decrease tasks_per_node 127 to 64
+    case ${CASE} in
+      "C768")
+        export tasks_per_node=64
+        ;;
+      *)
+        ;;
+    esac
+    ;;
+
+  "upp")
+    # decrease tasks_per_node 120 to 60
+    case ${CASE} in
+      "C768")
+        export tasks_per_node=60
+        ;;
+      *)
+        ;;
+    esac
+    ;;
+
+  "fcst")
+    # increase WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_{GDAS,GFS}
+    case ${CASE} in
+      "C768")
+        export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GDAS=20
+        export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS=25
+        (( WRTTASK_PER_GROUP_PER_THREAD_GDAS = WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GDAS * 6 ))
+        (( WRTTASK_PER_GROUP_PER_THREAD_GFS = WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS * 6 ))
+        export WRTTASK_PER_GROUP_PER_THREAD_GDAS
+        export WRTTASK_PER_GROUP_PER_THREAD_GFS
+        (( ntasks_quilt_gdas = WRITE_GROUP_GDAS * WRTTASK_PER_GROUP_PER_THREAD_GDAS ))
+        (( ntasks_quilt_gfs = WRITE_GROUP_GFS * WRTTASK_PER_GROUP_PER_THREAD_GFS ))
+        export ntasks_quilt_gdas
+        export ntasks_quilt_gfs
```

Comment:
@aerorahul Since this is duplicating calculations from config.ufs, I wonder if these should be moved from config.ufs to config.resources OR if we should allow the basic variables (i.e. `WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_${RUN}`) to be predefined in `config.ufs`.  For instance, changing

https://github.com/NOAA-EMC/global-workflow/blob/a42c833abe81df112e80da35843d698677d58bd7/parm/config/gfs/config.ufs#L283-L286

to

```bash
          export WRITE_GROUP_GDAS=${WRITE_GROUP_GDAS:-2}
          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GDAS=${WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GDAS:-15}
          export WRITE_GROUP_GFS=${WRITE_GROUP_GFS:-4}
          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS=${WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS:-20}
```

Then, instead of re-sourcing config.resources, Alex would re-source config.ufs, then config.resources.  What do you think?

Reply:
@WalterKolczynski-NOAA as well [^^^](https://github.com/NOAA-EMC/global-workflow/pull/2990/files#r1795325505).

Reply:
Morning @WalterKolczynski-NOAA @aerorahul C5 is down probably through tomorrow (Wed) but just wanted to check if you had a chance to look at this. I think the less we have to calculate in config.resources.$MACH the better. Thanks.

Reply:
I favor the second approach of allowing overrides, but want to hear from @aerorahul

Reply:
@DavidHuber-NOAA's recommendation makes sense.

Reply:
Thanks @WalterKolczynski-NOAA and @aerorahul. Hello @DavidHuber-NOAA ...My hands are tied with the C5 OS upgrade, but I was looking through this and writing out all the steps..Everything looks good for your second option until the re-sourcing of config.ufs.  In config.fcst, config.ufs is sourced via `source "${EXPDIR}/config.ufs" ${string}` where string is configured based on which model components are used, i.e., ocn, ice, wave, etc.  In order to get the string option into config.ufs, I believe I should re-source **config.fcst** which will do the re-sourcing of config.ufs. Let me know your thoughts. Thanks.

Reply:
@DavidBurrows-NCO Yes, you are correct about sourcing config.fcst -- that's the way to go about it rather than config.ufs.